bot.py

```
@@ -1,36 +1,107 @@
-
-@bot.hybrid_command(name="hello", description="Says hello to the user")
-async def hello(ctx):
-    await ctx.send(f"Hello {ctx.author.name}! 😃")
-
-# Run the bot
-if __name__ == "__main__":
-    token = os.getenv('DISCORD_TOKEN')
-    if not token:
-        raise ValueError("No token found. Make sure DISCORD_TOKEN is set in your environment variables.")
-    bot.run(token)
+require('dotenv').config();
```

Ruff (0.15.2)

[error] Statement ends with an unnecessary semicolon

(E703)